addons/skin.estuary/1080i/DialogVideoInfo.xml

```diff
@@ -46,488 +70,471 @@
-					<param name="visible" value="!IsEmpty(ListItem.Premiered)" />
-				</include>
-				<include name="InfoDialogMetadata">
-					<param name="control_id" value="155" />
-					<param name="label" value="[COLOR button_focus]$LOCALIZE[31048]: [/COLOR]$INFO[ListItem.Season,, $LOCALIZE[36905]]$INFO[ListItem.Episode, (, $LOCALIZE[20453])]" />
-					<param name="visible" value="!IsEmpty(ListItem.Season) + !Stringcompare(ListItem.DBType,episode)" />
-				</include>
-				<include name="InfoDialogMetadata">
-					<param name="control_id" value="156" />
-					<param name="label" value="[COLOR button_focus]$LOCALIZE[31017]: [/COLOR]$INFO[ListItem.Mpaa]" />
-					<param name="visible" value="!IsEmpty(ListItem.Mpaa)" />
-				</include>
-				<include name="InfoDialogMetadata">
-					<param name="control_id" value="157" />
-					<param name="label" value="[COLOR button_focus]$LOCALIZE[20141]: [/COLOR]$INFO[ListItem.Set]" />
-					<param name="visible" value="!IsEmpty(ListItem.Set)" />
-				</include>
-			</control>
-			<control type="image">
-				<left>17</left>
-				<top>185</top>
-				<width>1210</width>
-				<height>283</height>
-				<aspectratio>scale</aspectratio>
-				<texture colordiffuse="A0FFFFFF">colors/black.png</texture>
-			</control>
-			<control type="textbox">
-				<left>17</left>
-				<top>185</top>
-				<width>1210</width>
-				<height>283</height>
-				<align>center</align>
-				<aligny>center</aligny>
-				<font>font24_title</font>
-				<label>$LOCALIZE[31079]</label>
-				<visible>Integer.IsEqual(Container(50).NumItems,0) + IsEmpty(ListItem.Art(tvshow.banner)) + IsEmpty(ListItem.Art(banner))</visible>
-			</control>
-			<control type="image">
-				<left>77</left>
-				<top>185</top>
-				<width>1090</width>
-				<height>283</height>
-				<fadetime>150</fadetime>
-				<aspectratio>keep</aspectratio>
-				<texture>$VAR[BannerVar]</texture>
-				<visible>Integer.IsEqual(Container(50).NumItems,0)</visible>
-			</control>
-			<control type="list" id="50">
-				<top>158</top>
-				<width>1235</width>
-				<height>370</height>
-				<onleft>50</onleft>
-				<onright>50</onright>
-				<onup condition="System.hasAddon(script.extendedinfo) + Control.HasFocus(50)">RunScript(script.extendedinfo,info=extendedactorinfo,name=$INFO[Container(50).ListItem.Label])</onup>
-				<onup condition="!System.hasAddon(script.extendedinfo)">InstallAddon(script.extendedinfo)</onup>
-				<ondown>140</ondown>
-				<scrolltime>200</scrolltime>
-				<orientation>horizontal</orientation>
-				<itemlayout height="370" width="245">
-					<control type="group">
-						<top>10</top>
-						<control type="image">
-							<top>0</top>
-							<width>264</width>
-							<height>317</height>
-							<texture>DefaultActor.png</texture>
-							<aspectratio aligny="center">scale</aspectratio>
-							<bordertexture border="20">overlays/shadow.png</bordertexture>
-							<bordersize>20</bordersize>
+				<control type="list" id="50">
```

Comment:
will check again, i think intention was to reduce actor thumb loading times on first display. Not sure if it helps though.